torch/_inductor/cpu_vec_isa.py

```diff
@@ -306,9 +320,21 @@ def _check_and_append_supported_isa(
 
     return supported_isa
 
+def _is_arm_neoverse_v1() -> bool:
+    # reference: https://github.com/ARM-software/ComputeLibrary/blob/main/src/common/cpuinfo/CpuModel.cpp
+    
+    with open("/proc/cpuinfo") as _cpuinfo:
+        line = _cpuinfo.readline()
+        while line:
+            is_v1 = re.match(r"^CPU\spart(.*):\s0xd40(\n)$", line)
+            if is_v1:
+                return True
+            line = _cpuinfo.readline()
```

Comment:
Please wrap this code with `try: except:` to return false if cpuinfo is unaccessible for some reason